packages/shared/src/components/checklist/ChecklistStep.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { IconSize } from '../Icon';
+import ArrowIcon from '../icons/Arrow';
+import { ChecklistStepProps } from '../../lib/checklist';
+import { QuaternaryButton } from '../buttons/QuaternaryButton';
+import ChecklistAIcon from '../icons/ChecklistA';
+
+const ChecklistStep = ({
+  className = {},
+  step,
+  checked = false,
+  active = false,
+  onToggle,
+  children,
+}: ChecklistStepProps): ReactElement => {
+  const isCompleted = !!step.action.completedAt;
+  const isOpen = !isCompleted && checked;
+
+  return (
+    <div className={className.root}>
+      <button
+        type="button"
+        disabled={isCompleted}
+        className="flex justify-between items-center w-full"
+        onClick={(event) => {
+          event.preventDefault();
+
+          onToggle(step.action);
+        }}
+      >
+        <div className="flex flex-1 gap-1 items-center flex-start">
+          <QuaternaryButton
+            className={classNames(
+              '-ml-2 text-theme-label-tertiary',
+              isCompleted && 'text-theme-label-quaternary',
+            )}
+            id={step.action.type}
+            disabled={isCompleted}
+            icon={
+              <div
+                className={classNames(
+                  active && 'p-1 rounded-full bg-theme-bg-cabbage-opacity-24',
```

Comment:
Same here is this one not `cabbage-opacity-24`?

Reply:
I do not see this class existing

Reply:
That the new one you introduced in this PR no?

Reply:
Yeah, it is called `bg-theme-bg-cabbage-opacity-24`, do you want me to rename or?